models/lorenz_96_2scale/model_mod.f90

```diff
@@ -436,7 +436,8 @@ subroutine nc_write_model_atts(ncid, domain_id)
 integer :: i, xs, xe, ys, ye
 integer(i8) :: indx
 type(location_type) :: lctn
-real(r8) :: loc
+
+real(r8), allocatable :: xloc(:),yloc(:)
```

Comment:
Rather than make repeated calls to a 1d put_var, I decided to fill the array and make a single call to the 1d put_var.

Reply:
this is bitwise with Manhattan & does not change the runtime (unscientific test, it is faster 25.4s vs 25.9s).